crates/bevy_ecs/src/reflect/entity_commands.rs

```diff
@@ -466,4 +477,40 @@ mod tests {
 
         assert_eq!(world.entity(entity).get::<ComponentA>(), None);
     }
+
+    #[test]
+    fn insert_reflect_component_or_bundle() {
```

Comment:
We already have a test that inserts a component. I would add a test that just adds bundle.
I don't think you need a tests that adds both.

Reply:
I'll fix it immediately. I think I was too focused on my test code.

Reply:
Just to clarify: a separate test that only inserts a bundle.